### DISCUSSION TITLE
Indicator of changed settings

### DISCUSSION THREAD

**Discussion by sulaiban** [OrcaSlicer/OrcaSlicer#7827] [Ideas]
Hello Orca specialists 

In the Orca Slicer print settings, we change specific parameters to achieve our desired results in a project.
If you save this print profile as a user-defined profile, the colour coding is omitted compared to the basic profile. 
If you then reuse this profile, 
Not all changes are immediately recognisable.
I would like to be able to select that changed settings are highlighted in colour compared to the basic profile.

**Comment by LJKraus** [comment]
Colour currently only indicates unsaved profile changes. Adding a colour for each additional save could create a rainbow.

The Compare presets tool is a convenient way to see only the differences between two profiles.
![image](https://github.com/user-attachments/assets/cccfc6b3-3b55-40af-9890-9a83085e16af)
